pallets/manta-pay/src/rpc.rs

```diff
@@ -16,7 +16,7 @@
 
 //! MantaPay RPC Interfaces
```

Comment:
Conceptual question: Should this file better be part of the node/src/rpc path instead?

This is all client code as far as i can tell, where a `pallet` is supposed to be a runtime module

Reply:
I think it's more like a convention from substrate, check this example
https://github.com/paritytech/substrate/tree/master/frame/transaction-payment/rpc

Once we have more our own rpc methods, I'm fine if we decouple rpc code from pallet.

Reply:
Ok, makes sense. That pallet also interestingly decouples the runtime_api into its own subfolder of /rpc/
https://github.com/paritytech/substrate/tree/master/frame/transaction-payment/rpc/runtime-api

We don't need to do this now since we only have one (not yet released) version of this API, but it's good to keep in mind for future changes